home.asm

```diff
@@ -63,15 +63,33 @@ INCLUDE "home/vba.asm"
 INCLUDE "home/vwf.asm"
 
 ; data used within home
+SECTION "Alternate music table", ROM0
```

Comment:
Why? Technically a lot more of ROM0 could be single-table or few-routine sections, but would be clutter to do that.

Reply:
So the linker can sort them into gaps in home

Not attached to the change, just seemed like a good way to reduce work packing home with with minimal clutter

Reply:
Yeah, please just leave this out, it's unrelated to the PR title.